app/controllers/matchups_controller.rb

```diff
@@ -111,9 +102,7 @@ def matchups_for_round
 
   def generate_matchups(round, players)
     # generate pairings
-    active_players = players.select { |player| player.active == true }
-    inactive_players = players.select { |player| player.active == false }
-    pairings = Swissper.pair(active_players, delta_key: :win_count)
+    pairings = Swissper.pair(players.active, delta_key: :win_count)
```

Comment:
Just a comment - I believe the delta_key does not do anything right now since Player is not a Swissper object - idk if you are going to be using Swissper at all eventually, but thought you should know

Reply:
oh right, I am planning to replace swissper with my own implementation that is based on tsh's algorithms but I can fix the call to swissper as well if you like and keep it as another pairing option